config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml

```diff
@@ -18,3 +18,28 @@ presubmits:
         env:
         - name: NO_DOCKER
           value: "1"
+  - name: pull-service-catalog-xbuild
+    agent: kubernetes
+    context: pull-service-catalog-xbuild
+    always_run: true
+    skip_report: false
+    rerun_command: "/test pull-service-catalog-xbuild"
+    trigger: "(?m)^/test( all|pull-service-catalog-xbuild)\\s*"
```

Comment:
lol, `/test (all|pull-service-catalog-xbuild)`

Reply:
how did that space get in there, 😞